### ISSUE TITLE
is this repo dead

### ISSUE THREAD

**Issue by luki06** [pofider/phantom-html-to-pdf#105]
hi there is some module in this repo has vulnerability and need to be updated.

**Comment by bjrmatos** [comment]
we have similar issued reported [here](https://github.com/pofider/phantom-html-to-pdf/issues/69). 

the TL;DR is that those vulnerabilities are false positives in the scope of this package (unless you prove that it is not), we say this because Node Security reports (not sure if you are using that tool or another to find the vulnerabilities) only check for the presence of the package, but it does not check how that package is used. we don't see how the vulnerabilities reported can be used to exploit something in our case. most of the things reported are only used during installation time, and others comes from using phantomjs 1.9.8 (which we don't plan to upgrade to v2 by default because it has breaking changes, [however you can still use it in your app just fine](https://github.com/pofider/phantom-html-to-pdf#phantomjs2))
